frontend/src/muralForm/MuralForm.tsx

```diff
@@ -8,14 +8,13 @@ import {
 import { makeStyles, createStyles, Theme } from "@material-ui/core/styles";
 import AddressSearch from "../AddressSearch/addressSearch";
 import MultiAdd from "../multiAdd/MultiAdd";
-import ArtistSearchBar from "../ArtistSearch/ArtistSearch";
-import BoroughSearchBar from "../BoroughSearch/BoroughSearch";
 import ActionButtons from "../ActionButtons/ActionButtons";
 import EditIcon from "@material-ui/icons/Edit";
 import axios from "axios";
-import { CREATE_MURAL_API } from "../constants/constants";
+import { CREATE_MURAL_API, GET_ALL_ARTISTS_API, GET_ALL_BOROUGH_API } from "../constants/constants";
 import Alert from "@material-ui/lab/Alert";
 import ImageUpload from '../ImageUpload/ImageUpload'
+import ArtistBoroughSearch from "RegionSearch/ArtistBoroughSearch";
```

Comment:
how about putting this in `components/` (I'm going to migrate everything there eventually) and naming it something like `DropdownSearch` ?

Reply:
sounds good I ll do it

Reply:
updated